### PR TITLE
Add native tools bin folder to path on completion

### PR DESCRIPTION
**Pull request by jonfortescue**
Quick PR which adds the bin folder created by native tools to the path.

### CODE REVIEW

File: eng/common/init-tools-native.ps1 [hunk 1]
```diff
@@ -112,6 +112,7 @@ try {
   }
   if (Test-Path $InstallBin) {
     Write-Host "Native tools are available from" (Convert-Path -Path $InstallBin)
+    Write-Host "##vso[task.prependpath]" (Convert-Path -Path $InstallBin)
```
Comment (chcosta) [inline eng/common/init-tools-native.ps1]: Condition this based on whether Build.BuildNumber is defined so that this only runs in the VSTS context.

Comment (jonfortescue) [reply eng/common/init-tools-native.ps1]: Okay I can do that.

File: eng/common/init-tools-native.ps1 [hunk 2]
```diff
@@ -112,6 +112,9 @@ try {
   }
   if (Test-Path $InstallBin) {
     Write-Host "Native tools are available from" (Convert-Path -Path $InstallBin)
+    if (Test-Path env:BUILD_BUILDNUMBER) {
```
Comment (chcosta) [inline eng/common/init-tools-native.ps1]: I'm not certain if this is the correct way to determine if an environment variable is defined or not in Powershell.  Have you tested this?  StackOverflow mentions another method.

Comment (jonfortescue) [reply eng/common/init-tools-native.ps1]: I did test this and it worked haha. But I can do Alex's way as well.

Comment (chcosta) [reply eng/common/init-tools-native.ps1]: Cool.  My point was that if you tested this and it worked, then it's fine by me.  Alex's way is also fine.  Thanks @jonfortescue

**Pull request merged**